src/core/unit.c

```diff
@@ -1950,7 +1950,7 @@ void unit_trigger_notify(Unit *u) {
                         UNIT_VTABLE(other)->trigger_notify(other, u);
 }
 
-void unit_notify(Unit *u, UnitActiveState os, UnitActiveState ns, bool reload_success) {
+void unit_notify(Unit *u, UnitActiveState os, UnitActiveState ns, bool reload_success, bool stop_success) {
```

Comment:
> Functions with many bool params are so hard to call, since you never know which bool param you are actually passing there...

Reviewers: just say the word, and I'll look at replacing the bool params with unit virtual methods `bool stop_failed()` and `bool reload_failed()`.  (Or preferred alternative.  FYI, the specific values of `stop_result` and `reload_result` are never used, only their truthiness, and I wasn't planning to change that).